### PR TITLE
html-syntax

### PR DESCRIPTION
**Pull request by jghil**
html-syntax

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Favorite Pokemon</title>
+  </head>
+  <body>
+    <img src="https://64.media.tumblr.com/e5820cb0a101de8b6f904da68a10f7a6/8126c846a4fa53f1-46/s500x750/c155c9c40fd6f31962c9263febad3b8e33412c13.png">
```
Comment (BrettAlbright) [inline html-syntax/index.html]: make sure to include the alt text!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one tiny update needed!

File: html-syntax/index.html (hunk 1 above)
Comment (jghil) [reply html-syntax/index.html]: gotcha. included the alt text!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**